conf.ts

```diff
@@ -79,3 +79,8 @@ export function getConfig(): Config {
   };
   return conf;
 }
+
+/** @internal for test */
+export function resetConfig(newConf?: Config): void {
+  conf = newConf;
+}
```

Comment:
Please export it like

```ts
export const _internal = { resetConfig };
```

so that user who don't read document can recognize.

Reply:
`conf.ts` module is not exported in *deno.jsonc*.
So user cannot import it like `import ... from "jsr:@denops/test/conf";`
Will v2.0.0 be released in `deno.land/x`?

Reply:
Even so, it makes sense to add `_internal` since the developer (future me) might make a mistake.

Reply:
Fixed!